src/temperature/tmpSensor.h

```diff
@@ -3,23 +3,40 @@
 #include "max31725_cpp.h"
 #include <stdint.h>
 
+/**
+ * @brief The tmpSensor class interacts with a MAX31725 temperature sensor by binding it to another MAX31725
+ * sensor. It provides methods to confirm successful configuration register writing and geting sensor temperature data.
+ */
```

Comment:
Relevant design pattern is a wrapper/adapter.  See https://refactoring.guru/design-patterns/adapter